app.js

```diff
@@ -20,8 +22,11 @@ app.use(express.urlencoded({
 
 app.use(cors({ origin: 'http://localhost:3000', credentials: true }));
```

Comment:
The `origin` should be changed to the environment variable, more precisely to `process.env.FRONTEND_URL`.